examples/two_reservoir/two_reservoir_moea.py

```diff
@@ -43,19 +43,14 @@ def platypus_main(harmonic=False):
     wrapper = PlatypusWrapper(get_model_data(harmonic=harmonic))
 
     with platypus.ProcessPoolEvaluator() as evaluator:
-        algorithm = platypus.NSGAII(wrapper.problem, population_size=50, evaluator=evaluator)
+        algorithm = platypus.NSGAIII(wrapper.problem, population_size=50, evaluator=evaluator, divisions_outer=12)
```

Comment:
Any particular reason for switching the algorithm?

Reply:
No.